tools/app-toolkit/.gitignore

```diff
@@ -0,0 +1,3 @@
+build
```

Comment:
This apparently does not work, the build folder is included in the commits.

Reply:
Now it should be fine.